docs/resources/samples/range-basics.md

```diff
@@ -0,0 +1,280 @@
+---
+title: 'Range basics in Office Scripts'
+description: 'Learn basics about using the Range object in Office Scripts.'
+ms.date: 02/26/2021
+localization_priority: Normal
+---
+
+# Range Basics
```

Comment:
This is a good article to merge some of the existing samples into (or vice versa).

Reply:
Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)